source/documentation/upgrade_guide/topics/installing_virtual_product_hypervisors_from_local_repository.adoc

```diff
@@ -0,0 +1,88 @@
+[id="installing_virtual_product_hypervisors_from_local_repository_{context}"]
+= Installing {virt-product-shortname} hypervisors from a local repository
+
+If your system uses a private {virt-product-fullname} ({virt-product-shortname}) environment, but without Red Hat Satellite, you might need to install {virt-product-shortname} hypervisors (RHV-H) from a repository hosted on a local RHEL system instead of the Red Hat hosted Content Delivery Network (CDN).
+
+.Procedure
```

Comment:
Leave a blank line between header and steps. Otherwise, the rendering can be affected at a later date.